web/app/home/config.js

```diff
@@ -28,6 +28,7 @@ config = {
     home_people: { 
         name : 'home_people',
         url  : app.context + '/users',
+        method: app.apiMethod('POST'),
```

Comment:
I am sorry if I will sound a little critical, but I hope it is constructive. I think that app.apiMethod - does not deserver to exists. It is already too simple to set 'GET' or 'POST' and this method definitely makes it more complex. This can already be done with $.ajaxSetup, see https://api.jquery.com/jquery.ajaxsetup/